contracts/Pool.sol

```diff
@@ -426,24 +481,14 @@ contract Pool is PoolFDT {
     */
     function _transfer(address from, address to, uint256 wad) internal override {
         _whenProtocolNotPaused();
-        PoolLib.prepareTransfer(
-            withdrawCooldown,
-            depositDate,
-            totalCustodyAllowance[from],
-            balanceOf(from),
-            from,
-            to,
-            wad,
-            _globals(superFactory),
-            balanceOf(to),
-            recognizableLossesOf(from),
-            lockupPeriod
-        );
+        _checkTransfer(to, from, wad);
+        _updateDepositDate(wad, to);   
```

Comment:
Greatly reduced arguments for `_checkTransfer` given we no longer need to pass storage refs, and other now locally available variables.